app/controllers/simplified-account/settings/worldpay-details/credentials/worldpay-credentials.controller.js

```diff
@@ -0,0 +1,51 @@
+const { response } = require('@utils/response')
+const formatSimplifiedAccountPathsFor = require('@utils/simplified-account/format/format-simplified-account-paths-for')
+const paths = require('@root/paths')
+const { body, validationResult } = require('express-validator')
+const formatValidationErrors = require('@utils/simplified-account/format/format-validation-errors')
+
+function get (req, res) {
+  return response(req, res, 'simplified-account/settings/worldpay-details/credentials', {
+    backLink: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.worldpayDetails.index,
+      req.service.externalId, req.account.type)
+  })
+}
+
+const worldpayCredentialsValidations = [
+  body('merchantCode').not().isEmpty().withMessage('Enter your merchant code').bail()
+    .custom((value, { req }) => {
+      if (req.account.allowMoto && !value.endsWith('MOTO') && !value.endsWith('MOTOGBP')) {
+        throw new Error('Enter a MOTO merchant code. MOTO payments are enabled for the account')
+      }
```

Comment:
This needs to return `true` if the merchant code is valid, otherwise the validation fails with a generic error

Reply:
OK I can add that in